src/FileProviders/Embedded/src/Manifest/EmbeddedFilesManifest.cs

```diff
@@ -1,4 +1,4 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
😕 It looks like there are several encoding changes in this pull request. Can you confirm that this pull request is submitting files in the UTF-8 encoding _with BOM_?

Reply:
Current repo standard.